catalogd/internal/storage/localdir_test.go

```diff
@@ -169,15 +173,45 @@ func TestLocalDirStoraget(t *testing.T) {
 }
 
 func TestLocalDirServerHandler(t *testing.T) {
+	testMultiLineData := `{
+		"defaultChannel": "stable-v6.x",
+		"name": "cockroachdb",
+		"schema": "olm.package"
+	  }
+	  {
+		"entries": [
+		  {
+			"name": "cockroachdb.v5.0.3"
+		  },
+		  {
+			"name": "cockroachdb.v5.0.4",
+			"replaces": "cockroachdb.v5.0.3"
+		  }
+		],
+		"name": "stable-5.x",
+		"package": "cockroachdb",
+		"schema": "olm.channel"
+	  }
+	  `
+
```

Comment:
Can we use the fs.FS we're getting from `createTestFS` rather than creating a new variable for this test? 

I think we'll essentially verify jsonl formatting if we change 

```
return &fstest.MapFS{
		"bundle.yaml":  {Data: []byte(testBundle), Mode: os.ModePerm},
		"package.yaml": {Data: []byte(testPackage), Mode: os.ModePerm},
		"channel.yaml": {Data: []byte(testChannel), Mode: os.ModePerm},
	}
```

to be a contain a single file instead 

```
return &fstest.MapFS{
		"test-catalog.yaml":  {Data: generateJSONLines(_from testBundle, testPackage, testChannel_), Mode: os.ModePerm},
	}
```

Reply:
Context: our actual fs.FS in production also are singular files with multi line json blobs, instead of multiple files with individual json blobs. So tests mimicking that is more appropriate.